skorch/tests/test_setter.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for virtual parameter setters"""
+from unittest.mock import Mock
+
+import pytest
+
+
+def net_dummy():
```

Comment:
why not a fixture (perhaps move to conftest)

Reply:
@fixture done.
@move I think these functions are very specific to the test case. I'll move it to the test class instead.